chia/util/keyring_wrapper.py

```diff
@@ -113,11 +113,11 @@ def __init__(self, keys_root_path: Path = DEFAULT_KEYS_ROOT_PATH, force_legacy:
         if force_legacy:
             legacy_keyring = get_legacy_keyring_instance()
             if check_legacy_keyring_keys_present(legacy_keyring):
-                self.keyring = legacy_keyring
+                self.legacy_keyring = legacy_keyring
```

Comment:
I have some recollection that we used to assign the legacy keyring to `self.keyring` on purpose. If you've tested this all, then I'm good with these changes.